test/Dockerfile-centos7

```diff
@@ -1,5 +1,6 @@
 FROM centos:7
 
+RUN yum -y install epel-release
```

Comment:
its not possible to run this without EPEL?

Reply:
Missing python dependencies.